js/adapt-scrollPrompt.js

```diff
@@ -3,15 +3,26 @@ define([
   './scrollPromptView'
 ], function(Adapt, ScrollPromptView) {
 
-  Adapt.on('menuView:ready pageView:ready', function(view) {
+  Adapt.on('menuView:ready pageView:ready componentView:postRender', function(view) {
 
     var model = view.model;
```

Comment:
eslint reporting: unexpected var, use let or const instead (reported for all instances of var)

Reply:
Fixed in [775b1b5](https://github.com/cgkineo/adapt-scrollPrompt/pull/4/commits/775b1b578626b42bcb85520f605a1a84f019cec5)